api/src/YoutubeClient.cpp

```diff
@@ -29,6 +30,16 @@ std::string VideosResult::Video::GetMediumThumbnail() const {
   return json::StringRef(data_["snippet"]["thumbnails"]["medium"]["url"]);
 }
 
+bool VideosResult::Video::IsNSFW() const {
```

Comment:
Is this fine or should I also check for ytRating just to be safe.. that sounds like a good idea tbh

Reply:
the property either needs to be required or checked with `HasMember`